src/app/utils/ServerApiClient.js

```diff
@@ -51,12 +55,11 @@ export function markNotificationRead(account, fields) {
 }
 
 let last_page, last_views, last_page_promise;
-export function recordPageView(page, ref) {
+export function recordPageView(page, ref, account) {
     if (last_page_promise && page === last_page) return last_page_promise;
-    if (window.ga) { // virtual pageview
```

Comment:
please don't neuter GA here - this code should be removed only ifwhen GA is removed entirely.

please try to make PRs single-issue.  if you want to remove GA at the same time, make a second PR that does just that.  by adding overseer collection and removing GA, it makes the title of the PR misleading.

Reply:
Ok, I'll put it back